shell/browser/protocol_registry.cc

```diff
@@ -72,11 +75,18 @@ void ProtocolRegistry::RegisterURLLoaderFactories(
     DCHECK(!base::Contains(*factories, url::kFileScheme));
     factories->emplace(url::kFileScheme, AsarURLLoaderFactory::Create());
   } else if (type == URLLoaderFactoryType::kDocumentSubResource) {
-    // Only support requesting file:// subresource URLs when Chromium does so,
-    // it is usually supported under file:// or about:blank documents.
+    // Only support requesting file:// subresource URLs when:
+    // 1. it is supported by Chromium browser, such as under file:// or
+    //    about:blank documents;
+    // 2. web security is turned off.
```

Comment:
this is (1) OR (2), right?
```suggestion
    // 1. it is supported by Chromium browser, such as under file:// or
    //    about:blank documents; or
    // 2. web security is turned off.
```